src/components/atoms/pageTitle/PageTitle.tsx

```diff
@@ -16,7 +16,7 @@ export function PageTitle({
   fullWidth: _fullWidth,
 }: PageTitleProps) {
   const titleUnderline: SxProps = {
-    marginBottom: '1rem',
+    paddingBottom: '8px',
```

Comment:
I think we need to have a margin and padding.

Although the border is now properly away from the title, it's too close to the text after it.

Also
```suggestion
    paddingBottom: '0.5rem',
```

Reply:
YES! Sorry, overlooked that.

Reply:
<img width="437" alt="Screenshot 2025-03-14 at 8 01 04 PM" src="https://github.com/user-attachments/assets/9c270614-a49e-4b9d-9a18-2f1b2d0ee804" />

Reply:
^ threw the same 0.5rem in as `marginBottom`

Reply:
The figma has it as a full rem.